man/man1/fi_strerror.1

```diff
@@ -0,0 +1,25 @@
+.TH fi_strerror 1 "2016\-06\-30" "Libfabric Programmer\[aq]s Manual" "\@VERSION\@"
+.SH NAME
+.PP
+fi_strerror - display libfabric error strings
+.SH SYNOPSIS
+.IP
+.nf
+\f[C]
+fi_strerror\ FI_ERROR_CODE
+\f[]
+.fi
+.SH DESCRIPTION
+.PP
+Display the error string for the given numeric \f[C]FI_ERROR_CODE\f[].
+\f[C]FI_ERROR_CODE\f[] may be a hexadecimal, octal, or decimal constant.
+For convenience, the absolute value of \f[C]FI_ERROR_CODE\f[] will be
+used, since libfabric APIs typically return negative error codes, but
+\f[C]fi_strerror\f[](3) accepts only positive error codes.
```

Comment:
But that is a reference to section **3** of the man pages (the libfabric function call), not this helper utility.  If it's still confusing, can you suggest a rewording?

A separate enhancement that could be made would be to make the function call version also accept negative values, then there would be no discrepancy between the utility and the function.